testnet/stacks-node/src/event_dispatcher.rs

```diff
@@ -433,6 +436,13 @@ impl EventObserver {
             "pox_v3_unlock_height": pox_constants.v3_unlock_height,
         });
 
+        if let Some(signer_bitvec) = signer_bitvec_opt {
+            payload.as_object_mut().unwrap().insert(
+                "signer_bitvec".to_string(),
+                serde_json::to_value(signer_bitvec).unwrap_or_default(),
```

Comment:
I can't tell what the serialized JSON is for this. Is it a hex string?

Reply:
Yeah, check out the PR description (last sentence). It's a length-prefixed hex string

Reply:
Oh okay, thanks!